archive.php

```diff
@@ -5,7 +5,6 @@
 <div class="container">
 	<div class="row">
 		<div class="col-md-8">
-			<?php today_archive_pagination(); ?>
 			<?php if ( have_posts() ) : ?>
 				<?php echo today_post_list_display_feature( '', $posts, array( 'posts_per_row' => 3, 'layout' => 'vertical' ) ); ?>
 				<?php ucfwp_the_posts_pagination(); ?>
```

Comment:
`col-lg-4 pl-lg-5` would be ideal over `col-md-4` for the sidebar wrapper column.  Adding `mt-4 mt-lg-0` would also improve vertical spacing between the main content and collapsed sidebar at smaller screen sizes

Reply:
Also updated.